pinot-common/src/main/java/com/linkedin/pinot/common/utils/CommonConstants.java

```diff
@@ -289,6 +289,13 @@ public ServerType getServerType() {
       public static final String DOWNLOAD_URL = "segment.offline.download.url";
       public static final String PUSH_TIME = "segment.offline.push.time";
       public static final String REFRESH_TIME = "segment.offline.refresh.time";
+      /**
+       * This field will be populated only for use cases that are refreshing segments and
+       * have minion configured. It will contain COMMITTING + System.currentMillis(), time right
+       * before any permanent changes happen to the segment, aka, right before
```

Comment:
will be nice to have an example here. Maybe "COMMITTING-1513013204080" where time is in msecs since epoch? Better yet, something like:  "COMMITTING-20171210T09:31:55.765" ?

Reply:
The human-readable timestamp would be better. Make sure that it's in UTC time though, otherwise there are some edge cases around DST which you really don't want to deal with for this kind of feature.